server/seeds/songSeeds.json

```diff
@@ -1,14 +1,50 @@
 [
   {
-    "songId": "1",
-    "title": "Song1",
-    "artists": ["Band"],
+    "songId": "4gMgiXfqyzZLMhsksGmbQV",
+    "title": "Another Brick in the Wall, Pt. 2",
+    "artists": ["Pink Floyd"],
     "album": {
-      "title": "first album",
-      "image": "image.jpg"
+      "title": "The Wall",
+      "image": "https://i.scdn.co/image/ab67616d00001e02288d32d88a616b9a278ddc07"
     },
-    "link": "web.com",
-    "previewUrl": "fadfsggdsgsf.com",
-    "expiration": "2022-09-17T17:09:38Z"
+    "link": "https://api.spotify.com/v1/tracks/4gMgiXfqyzZLMhsksGmbQV",
+    "previewUrl": "https://p.scdn.co/mp3-preview/92210b01fcc6efa80008663c158baf79cf0d7b5f?cid=774b29d4f13844c495f206cafdad9c86",
+    "expiration": "2032-09-17T17:09:38Z"
+  },
+  {
+    "songId": "4cOdK2wGLETKBW3PvgPWqT",
+    "title": "Never Gonna Give You Up",
+    "artists": ["Rick Astley"],
+    "album": {
+      "title": "Whenever You Need Somebody",
+      "image": "https://i.scdn.co/image/ab67616d00001e025755e164993798e0c9ef7d7a"
+    },
+    "link": "https://api.spotify.com/v1/tracks/4cOdK2wGLETKBW3PvgPWqT",
+    "previewUrl": "https://p.scdn.co/mp3-preview/0b6da17f858f104337fac626c4bac972d3947564?cid=774b29d4f13844c495f206cafdad9c86",
+    "expiration": "2032-09-17T17:09:38Z"
```

Comment:
Expiration doesn't need to be set because there is already a .pre('save') hook that sets it automatically in Song.js.